torch/csrc/autograd/input_buffer.cpp

```diff
@@ -9,7 +9,7 @@ InputBuffer::InputBuffer(size_t size)
   : buffer(size)
   {}
 
-void InputBuffer::add(size_t pos, const Variable& var) {
+void InputBuffer::add(size_t pos, Variable var) {
```

Comment:
nit: it used to be an r-value reference. it seemed to be better than a regular value

Reply:
We were trying to signify that InputBuffer takes ownership of `var`, but the standard way to do that with smart shared pointers is just to take it by value:

See (e) in: https://herbsutter.com/2013/06/05/gotw-91-solution-smart-pointer-parameters/